tools/pkgconf/files/pkg-config

```diff
@@ -4,6 +4,8 @@ ${STAGING_DIR_HOST}/bin/pkg-config.real \
 --keep-system-cflags \
 --keep-system-libs \
 --define-variable=prefix="${STAGING_PREFIX}" \
+--define-variable=prefix_host="${STAGING_DIR_HOST}" \
+--define-variable=prefix_hostpkg="${STAGING_DIR_HOSTPKG}" \
```

Comment:
Are these standard variable names or something OpenWrt custom?

Reply:
OpenWrt custom but it's really caused by our buildsystem and glib2 including both host and target tools in the pkg-config file hence we need a way to reference hostpkg directories... Alternative is to provide meson override (but that will only apply to meson build system)